### ISSUE TITLE
This library is Not available in maven repo. What are the other options?

### ISSUE THREAD

**Issue by muvelus** [slytechs-repos/jnetpcap-api#9]
Getting the following error when added as a dependency in Maven.

`Could not find artifact com.slytechs.jnet.jnetpcap:jnetpcap-api:pom:0.10.0 in central (https://repo.maven.apache.org/maven2)
`

**Comment by SecureNetWizard** [comment]
Fixing. New binaries re-uploading to maven-central.

**Comment by ssill2** [comment]
I'm still not seeing the api artifact available in mvn central

**Comment by SecureNetWizard** [comment]
I have a new ticket open with the Sonata team. Will get this resolved. The packages have been uploaded.

Also I have major updates to the jnetruntime-api and jnetpcap-api modules and will make sure everything is uploaded by next week. Massive amount of work about to be released.

**Comment by ssill2** [comment]
I'm looking to evaluate this for a project and we use only the latest LTS releases of java.  Do you have builds that work with java 21?

**Comment by SecureNetWizard** [comment]
Yes we do. We support 21 LTS with JDK 21 specific packages as FFM was a "feature" and requires a command line flag and source 21 specific classfiles.

The JDK 22 packages will be binary compatible with all future JDK versions, as FFM is now a production feature in 22 and on.

**Comment by ssill2** [comment]
sounds good.  I'll wait until next week to dive into this then.  Previously when I did pcap stuff I had used pcap4j and that worked fine for what I was doing, but that project appears to have been abandoned and the new stuff I want to do will be higher volume, which is what lead me to your project.

**Comment by SecureNetWizard** [comment]
Sounds good. I'll keep everyone informed when the issue is resolved and packages are on maven central.

**Comment by JOZA-ORANGE** [comment]
> Sounds good. I'll keep everyone informed when the issue is resolved and packages are on maven central.

Excuse me, is everything ready now?

**Comment by SecureNetWizard** [comment]
Almost ready, we are very close.  So many moduales have to be aligned for this HUGE release. 

The functinality is all done, but the work has been mainly refactoring code that should not be public but is none the less part of the module. This is tightly controlled as once publicly released we're on the hook for supporting that API. Polishing up the API took a fair amount of time.

Let me present you the map of all of the modules that are being released one by one, starting with full `jnetpcap-sdk` and `protocol-sdk`  with various protocol packs.

https://github.com/orgs/slytechs-repos/discussions/3

**Comment by ssill2** [comment]
I'm hoping to get back into a project where I'll need this.  It still doesn't seem to be out on maven central repo yet.   Any estimates?
